packages/@aws-cdk/custom-resource-handlers/lib/custom-resources/aws-custom-resource-handler/index.ts

```diff
@@ -1,17 +1,277 @@
 /* eslint-disable @typescript-eslint/no-require-imports */
-export { PHYSICAL_RESOURCE_ID_REFERENCE } from './shared';
+/* eslint-disable no-console */
```

Comment:
No, this is here so that eslint doesn't complain about `console.log` throughout the file. This specific eslint exception covers the file from its position downward. I will make sure we remove any other `eslint-disable no-console` statements found in the rest of the file, though.